### PR TITLE
Fix #39 and #40, update version to 1.13.2, fix some typos in docs README

### LINKED ISSUE #39: Expected behavior of tweet.bio is to return empty string for a None, instead it returns None

### PR DESCRIPTION
**Pull request by fionapigott**
Fixed the date formatting issue and corrected the behavior of the bio getter to reflect the docs. I'll build the docs and push the new doc pages as soon as this gets merged onto master.

### CODE REVIEW

**Comment by fionapigott** [comment]
Also I'll push the changes to PyPI once this gets approved.

File: tweet_parser/getter_methods/tweet_user.py [hunk 1]
```diff
@@ -144,9 +144,13 @@ def get_bio(tweet):
     """
 
     if is_original_format(tweet):
-        return tweet["user"].get("description", "")
+        bio_or_none = tweet["user"].get("description", "")
     else:
-        return tweet["actor"].get("summary", "")
+        bio_or_none = tweet["actor"].get("summary", "")
+    if bio_or_none is None:
```
Comment (jrmontag) [inline tweet_parser/getter_methods/tweet_user.py]: I think with the default value of `""`, the `get()` method should never return None, correct? Have you seen situations where this `if` is needed?

Comment (fionapigott) [reply tweet_parser/getter_methods/tweet_user.py]: It returns "None" because sometimes there's a None value in the payload now. Like, {"summary": None}

Comment (fionapigott) [reply tweet_parser/getter_methods/tweet_user.py]: Was happening to me when I was getting data.

Comment (jrmontag) [reply tweet_parser/getter_methods/tweet_user.py]: Oh, I see. Cool, thanks for clarifying!

**Pull request merged**